CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * General
   * Added a Glean Health ping which collects telemetry health data into a single ping sent before and after initialization in order to track issues with Glean storage files and other telemetry health characteristics. ([#3221](https://github.com/mozilla/glean/pull/3221))
+  * Remove infill 0 buckets from custom distributions ([#3246](https://github.com/mozilla/glean/pull/3246))
```

Comment:
Teeeechnically this is breaking I guess, as [it was when we did it before](https://github.com/mozilla/glean/blob/11d6ad6930c2fcd797585a7e01514d0b20768b47/CHANGELOG.md?plain=1#L229).

But it's not breaking in the code, "just" the pipeline.